frontendActions/src/main/java/com/virtuslab/gitmachete/frontend/actions/BaseRebaseBranchOntoParentAction.java

```diff
@@ -90,23 +88,19 @@ public void update(AnActionEvent anActionEvent) {
 
   protected void doRebase(AnActionEvent anActionEvent, BaseGitMacheteNonRootBranch branchToRebase) {
     Project project = ActionUtils.getProject(anActionEvent);
-    Option<IGitMacheteRepository> gitMacheteRepository = getGitMacheteRepository(anActionEvent);
```

Comment:
If I'm right `DataKeys#KEY_GIT_MACHETE_REPOSITORY` from javadoc on top of file can be removed

Reply:
Done on another PR